src/users/users.service.ts

```diff
@@ -10,7 +10,7 @@
 import { Injectable, Logger } from '@nestjs/common';
 import { TokenExpiredError } from '@nestjs/jwt';
 import { InjectRepository } from '@nestjs/typeorm';
-import * as bcrypt from 'bcrypt';
+import * as bcrypt from 'bcryptjs';
```

Comment:
why this change?

Reply:
bcrypt is implemented as a c++ native library, and there is compatibility issue when installing on some platforms. bcryptjs is completely in js and won't cause these problems.